dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp

```diff
@@ -236,10 +236,22 @@
         %>
         <div class="wysiwyg-wrapper">
             <div id="<%=field.getVelocityVarName()%>aceEditor" class="classAce aceTall" style="display: none"></div>
-            <textarea <%= isReadOnly?"readonly=\"readonly\"":"" %>
-                    class="editWYSIWYGField aceText aceTall" 
-                    name="<%=field.getFieldContentlet()%>"
-                    id="<%=field.getVelocityVarName()%>"><%=UtilMethods.htmlifyString(textValue)%></textarea>
+
+                <style type="text/css">
+                   dot-asset-drop-zone {
+                     width: 100%;
+                     height: 91%;
+                     position: absolute;
+                     z-index: 1;
+                     pointer-events: none;
+                   }
+                </style>
+                <dot-asset-drop-zone id="dot-asset-drop-zone-<%=field.getVelocityVarName()%>"></dot-asset-drop-zone>
```

Comment:
why not the `dot-asset-drop-zone` wrap the textarea ?

Reply:
I understand this was its intended usage, however, when wrapping the textarea with it I had issues with pointer-events in the editor. If this is a problem please let me know.

Related https://github.com/dotCMS/core/blob/f6352f95de7842889b29490cca8246088ceedf13/dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field_js.jsp#L381-L393